### ISSUE TITLE
comment() not working with DBprefix.

### ISSUE THREAD

**Issue by manuzcom** [yajra/laravel-oci8#310]
```php
Schema::schema()->create('visuals', function ($table) {
        $table->increments('id')->comment('INDEX');
```
this is not working. 

\Schema\Comment.php line 90 should change like this.
```php
        $table = $this->wrapValue($table);
        $table = $this->connection->getTablePrefix() . $table; // [+] this line should add.
```

**Comment by yajra** [comment]
Looks like a valid issue. Can you please submit a PR to fix this? Thanks!

**Comment by yajra** [comment]
Fixed on #314
